tidy3d/components/autograd/derivative_utils.py

```diff
@@ -15,6 +15,11 @@
 FieldData = dict[str, ScalarFieldDataArray]
 PermittivityData = dict[str, ScalarFieldDataArray]
 
+__all__ = [
```

Comment:
I think I was shuffling things around and might have moved this out of habit. The convention I'm used to is to have this defined near the top of the file so it's easy to see which functions a module is supposed to export. I can change it back though since we usually have it at the bottom it seems.